### ISSUE TITLE
Uncaught TypeError: cell2 is null when running ToolBar story

### ISSUE THREAD

**Issue by tbouffard** [maxGraph/maxgraph#254]
When dragging an element of the toolbar to a swimlane, the following error is displayed in the console:
```
Uncaught TypeError: cell2 is null
    isLayer GraphDataModel.js:374
    getDropTarget SwimlaneMixin.js:280
    mouseMove SelectionHandler.js:759
    fireMouseEvent EventsMixin.js:640
    installListeners GraphView.js:1684
```
This was detected in #230 and it seems very closed to #175.

**Comment by tbouffard** [comment]
This may relates to #357 which has been fixed in v0.9.0.

**Comment by tbouffard** [comment]
No longer reproduced, so closing
